storybook/stories/Brand/Colors.stories.tsx

```diff
@@ -21,6 +22,31 @@ storiesOf("Brand/Colors", module)
             );
           })}
         </View>
+        <View style={{ flexDirection: "row", flexWrap: "wrap", backgroundColor: BACKGROUND_COLOR }}>
+          {[Colors.BLACK, Colors.BLACK_65, Colors.BLACK_45, Colors.BLACK_25, Colors.BLACK_06].map((color, index) => {
+            return (
+              <View
+                style={{ height: 60, margin: 8, width: "14.28%", backgroundColor: color }}
+                key={color + index.toString()}
```

Comment:
react keys only need to be locally unique within the collection, so i would expect that `color` and `index` are both sufficient.

sometimes i leave notes like this on pull requests because i want to share what i've learned. it's not a blocker or any kind of priority change, though.